spec/features/formats/publication_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe "Publication format" do
```

Comment:
I wonder if should mention the word schema in here. I assume we've already got precedence in news article schema so could well be a problem for another day. Could be "format using publication schema"

Reply:
Yeah, I followed the convention set in the news article test:
```
RSpec.describe "News article format" do
```

Reply:
Sure, can fix them both in another day. Not too big a deal.